WordPress/Classes/ViewRelated/Blog/Blog Dashboard/ViewModel/BlogDashboardViewModel.swift

```diff
@@ -111,6 +112,15 @@ private extension BlogDashboardViewModel {
         }
     }
 
+    func createSnapshot(from cards: [DashboardCardModel]) -> DashboardSnapshot {
+        let items = cards.map {DashboardItem.cards($0)}
```

Comment:
[nit]

```suggestion
        let items = cards.map { DashboardItem.cards($0) }
```
```suggestion
        let items = cards.map {DashboardItem.cards($0)}
```

Reply:
Fixed in dcf939a5085c7132aa79407026b4975b12f37208